src/main/java/com/example/bookshop/repository/OrderItemRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.example.bookshop.repository;
+
+import com.example.bookshop.model.OrderItem;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface OrderItemRepository extends JpaRepository<OrderItem, Long> {
```

Comment:
if you add `@Repository` in `OrderRepository` interf, you should add `@Repository` in `OrderItemRepository` for the consistency